DESCRIPTION

```diff
@@ -5,7 +5,7 @@ Authors@R: c(
     person("Roman", "Mukhin", email = "roman@ebookapplications.com", role = c("aut", "cre")),
     person("Evan", "Biederstedt", role = "aut"),
     person("Andrey", "Isaev", role = "aut"))
-Description: The package provides a function to convert AnnData h5ad files to Seurat5 object and saves it as an RDS file. The package was created especially for the Cell Annotation Project (CAP), so please test if it suits your needs. The package is written on clear R, no reticulate dependencies. The package currently supports the following AnnData fields: X, obs, var, raw.X, raw.var, obsm, uns.
+Description: The package provides functions to convert AnnData (.h5ad) files to Seurat v5 objects and saves them as RDS files. The package was created especially for the Cell Annotation Project (CAP), so please test if it suits your needs. It is written in pure R, with no reticulate dependencies. The package currently supports AnnData (.h5ad) files created via AnnData v0.10+ and converts the following fields: X, obs, var, raw.X, raw.var, obsm, and uns.
```

Comment:
The package was created especially for the [Cell Annotation Project](https://celltype.info/) (CAP), so please test if it suits your needs. --> 

The package was created specifically for the needs of the Cell Annotation Project (CAP), so please test it first to see if it meets your requirements